presto-docs/src/main/sphinx/sql/create-table-as.rst

```diff
@@ -36,9 +39,18 @@ Create a new table ``orders_by_date`` that summarizes ``orders``::
     FROM orders
     GROUP BY orderdate
 
+Create the table ``orders_by_date`` if it does not already exist::
+
+    CREATE TABLE IF NOT EXISTS orders_by_date
+    AS
```

Comment:
Put the `AS` on the previous line, like the example below (the first one is more complex because it has table properties)